### PR TITLE
Locations rspec refactor

### PR DESCRIPTION
**Pull request by amsmyth1**
Completed Usability section and test suite. 
Refactored all tests and added edge cases for boolean logic.
Installed orderly gem to use in spec tests for user story 16 and 18.
Added 27 tests.

Rspec passing
Simplecov coverage 100%

### CODE REVIEW

File: spec/features/location_bookstores/new_spec.rb [hunk 1]
```diff
@@ -1,33 +1,64 @@
...
+      name: "Joseph Fox BookStop",
+      inventory: 444,
+      open: false})
+
+    visit "/locations/#{location_1.id}/bookstores/new"
+
     fill_in("bs_name", with: bookstore_2.name)
     fill_in("bs_inv", with: bookstore_2.inventory)
-    check('open?')
+    uncheck('open?')
```
Comment (klatour324) [inline spec/features/location_bookstores/new_spec.rb:40]: Great use of checking edge cases here to make sure that you are returning false when it is not checked!

**Review by klatour324 — APPROVED** [review]
Great work here Alexa! Seriously I really saw the value of how you implemented the orderly gem and like where you tested your boolean logic to make sure the other side of the conditional made sense.
